js/src/setup-mc/setup-stepper/setup-accounts/index.js

```diff
@@ -89,6 +90,7 @@ const SetupAccounts = ( props ) => {
 		isReady: isGoogleMCReady,
 	} = useGoogleMCAccount();
 	const { hasFinishedResolution } = useGoogleAdsAccount();
+	const storeAddressSynced = useStoreAddressSynced();
```

Comment:
This hook cannot be used here until after the MC account is connected. Otherwise it will send an API request to `/gla/mc/contact-information` which will result in a 401 error.

<img width="671" alt="image" src="https://github.com/user-attachments/assets/e776613b-1af1-41dd-9dde-71253aa16edf">

Either we need to incorporate `useGoogleMCAccount()` into the `useStoreAddressSynced()` hook so that it returns `false` early if there is no MC account, or rely on a `useState()` value that gets updated after the verification is made.

Reply:
Thanks @joemcgill . Completely missed that 🤦 . I've updated the `useStoreAddressSynced` hook. Let me know what you think.